docs/_layouts/post.html

```diff
@@ -7,11 +7,11 @@
   <nav class="side-nav">
     <button id="toggleNavButton">Menu</button>
     <ul>
-      <li>
+      <li class="{% if page.url == '/guide.html' %}active{% endif %}">
```

Comment:
If we add `permalink: /guide/` to the guide.md front matter, we could use the actual `/guide/` url here.